one.css

```diff
@@ -225,6 +225,9 @@ Buttons
     border-radius: 50%;
     background-color: var(--orange);
     font-size: 1rem;
+    width: 2.5rem;
+    height: 2.5rem;
+    outline: none;
 }
 
 .btn-float:hover {
```

Comment:
maybe you can also change cursor to pointer on hover!

Reply:
done